src/Microsoft.ApplicationInsights.AspNetCore/Logging/Implementation/ApplicationInsightsLoggerProvider.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Microsoft.ApplicationInsights.AspNetCore.Logging
+{
+    using Microsoft.Extensions.Logging;
+
+    internal class ApplicationInsightsLoggerProvider : ILoggerProvider
```

Comment:
Why is all the ILogger stuff internal?
Shouldn't a customer be able to use it if so desired?
This is something we had on our roadmap to consider.